src/gui/nav/ViewSlider.ts

```diff
@@ -48,17 +46,32 @@ export class ViewSlider implements Component<ViewSliderAttrs> {
 	private _busy: Promise<unknown>
 	private _parentName: string
 	private _isModalBackgroundVisible: boolean
+	private resizeListener: windowSizeListener = () => this._updateVisibleBackgroundColumns()
+	private handleHistoryEvent = () => {
+		const prev = this.getPreviousColumn()
+		if (prev != null && prev.columnType !== ColumnType.Foreground) {
+			this.focusPreviousColumn()
```

Comment:
when we return `false`, `WindowFacade` will try to revert the history change but that should be okay because the item in the list has the same URL, right? In my (limited) testing it works except it fires `updateUrl()` twice.

out of scope but we probably should discuss changing how routing works for single-column layout so that if we have `mailListId/mailId` it shows you the viewer and if you are in the list it should be just `mailLlistId`.

Reply:
false means we handled it and the window facade should try to revert the change, yes